traffic_stats/build/traffic_stats.spec

```diff
@@ -75,8 +75,8 @@ godir=src/github.com/apache/incubator-trafficcontrol/traffic_stats/influxdb_tool
 ( mkdir -p "$godir" && \
   cd "$godir" && \
   cp -r "$TC_DIR"/traffic_stats/influxdb_tools/* . && \
-  go build sync_ts_databases.go
-  go build create_ts_databases.go
+  go build sync/sync_ts_databases.go
+  go build create/create_ts_databases.go
```

Comment:
It won't let me comment on line 69, but it needs to be removed since we are vendoring now.
`go_get_version "$oldpwd"/src/github.com/influxdata/influxdb v1.0.1 && \`